app/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt

```diff
@@ -36,6 +36,8 @@ interface Pixel {
         FORGET_ALL_PRESSED_BROWSING("mf_bp"),
         FORGET_ALL_PRESSED_TABSWITCHING("mf_tp"),
         FORGET_ALL_EXECUTED("mf"),
+        FORGET_ALL_AUTO_RESTART("mf_r"),
+        FORGET_ALL_AUTO_RESTART_WITH_INTENT("mf_ri"),
```

Comment:
any new pixels should be under the `m_` namespace. `mf` is a legacy one so don't worry about being consistent with that